### ISSUE TITLE
Option for disabling anchror hover previews

### ISSUE THREAD

**Issue by santhoshtr** [daviferreira/medium-editor#243]
For a use case where we handle links in a different way, we need an option to remove the link preview when user moves mouse over the link.

I removed this effect using
```
MediumEditor.prototype.editorAnchorObserver = function () {};
```
But prefer to have an option that allows this in a better way.
Thanks!

**Comment by nmielnik** [comment]
This was fixed by #478 .  You can now use the `disableAnchorPreview` option, or even override the anchor preview with your own anchor preview extension!

**Comment by santhoshtr** [comment]
Thanks!
